process/sandbox.md

```diff
@@ -91,3 +91,16 @@ Frequency of reviews can be found on the [repo's README under "Frequency"](https
 ### Annual review
 
 Once in the Sandbox, projects are subject to an [Annual Review](https://github.com/cncf/toc/blob/master/process/sandbox-annual-review.md).
+
+### Sandbox archival
```

Comment:
I feel like this should be a separate PR; folks are going to have feelings about archival, and I'd like to merge the milestones while that argument plays out.